plugins/inputs/azure_monitor/azure_monitor_test.go

```diff
@@ -612,51 +614,6 @@ func TestInit_NoSubscriptionID(t *testing.T) {
 	require.Error(t, am.Init())
 }
 
-func TestInit_NoClientID(t *testing.T) {
-	file, err := os.ReadFile("testdata/toml/init_no_client_id.toml")
-	require.NoError(t, err)
-	require.NotNil(t, file)
-	require.NotEmpty(t, file)
-
-	var am *AzureMonitor
-	require.NoError(t, toml.Unmarshal(file, &am))
-
-	am.Log = testutil.Logger{}
-	am.azureManager = &mockAzureClientsManager{}
-
-	require.Error(t, am.Init())
-}
-
-func TestInit_NoClientSecret(t *testing.T) {
-	file, err := os.ReadFile("testdata/toml/init_no_client_secret.toml")
-	require.NoError(t, err)
-	require.NotNil(t, file)
-	require.NotEmpty(t, file)
-
-	var am *AzureMonitor
-	require.NoError(t, toml.Unmarshal(file, &am))
-
-	am.Log = testutil.Logger{}
-	am.azureManager = &mockAzureClientsManager{}
-
-	require.Error(t, am.Init())
-}
-
-func TestInit_NoTenantID(t *testing.T) {
-	file, err := os.ReadFile("testdata/toml/init_no_tenant_id.toml")
-	require.NoError(t, err)
-	require.NotNil(t, file)
-	require.NotEmpty(t, file)
-
-	var am *AzureMonitor
-	require.NoError(t, toml.Unmarshal(file, &am))
-
-	am.Log = testutil.Logger{}
-	am.azureManager = &mockAzureClientsManager{}
-
-	require.Error(t, am.Init())
-}
-
```

Comment:
Why are these tests removed?

Reply:
The upstream part of the [lib](https://github.com/logzio/azure-monitor-metrics-receiver/pull/18) has also been removed.